spring-cloud-modules/spring-cloud-contract/spring-cloud-contract-producer/pom.xml

```diff
@@ -29,14 +29,24 @@
             <groupId>org.springframework.boot</groupId>
             <artifactId>spring-boot-starter-data-rest</artifactId>
         </dependency>
```

Comment:
As rest-assured is inherited and brings old groovy, exclude that groovy version (2.5.6)